### PR TITLE
Add "we're searching for our next CIO article"

### PR DESCRIPTION
**Pull request by ascott1**
This adds a brief article about our search for our next CIO.

## Review

@Scotchester @higs4281 @marteki

### CODE REVIEW

File: _posts/2016-04-22-next-cio.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+The [Consumer Financial Protection Bureau (CFPB)](http://consumerfinance.gov) is a government agency that helps consumer finance markets work for Americans by making rules more effective, by consistently and fairly enforcing those rules, and by empowering consumers to take more control over their economic lives.  From the Bureau’s founding in the wake of the 2008 financial crisis, it has always been clear that we need to leverage modern technology and take a data-driven approach to achieve these goals.  The CFPB technology team uses and produces [open source code](https://github.com/cfpb), has embraced systems in the public cloud, and follows an [Agile](http://www.agilemanifesto.org/) software development model. Our team is comprised of talented individuals from both the public and private sectors, and through CFPB’s T[Technology and Innovation Fellows](http://www.consumerfinance.gov/jobs/technology-innovation-fellows/) program, we bring together designers, developers, data scientists, and security professionals to collaborate on building CFPB technology.
```
Comment (higs4281) [inline _posts/2016-04-22-next-cio.md]: personal pet peeve: "is comprised of" is a misuse of the word. 
either "Our team comprises talented individuals ..." or "Our team is made up of talented individuals ..."

**Comment by higs4281** [comment]
👍 with one nit.

File: _posts/2016-04-22-next-cio.md (hunk 1 above)
Comment (marteki) [inline _posts/2016-04-22-next-cio.md]: Extra T before the link: "CFPB’s T[Technology and Innovation Fellows](http://www.consumerfinance.gov/jobs/technology-innovation-fellows/)"

Comment (Scotchester) [inline _posts/2016-04-22-next-cio.md]: @higs4281 That's not a pet peeve, that's just looking out for correct usage :D

**Comment by marteki** [comment]
Putting on my FEWD hat, I'd ask for semantic line breaks in the code. 😄  :smile: :smile:

Putting on my English major hat, I pointed out an errant T in the diff. I also wish for shorter and less complicated sentence constructions, but I understand the constraints and why that isn't happening for this article.

Basically, take charge of the extra T, and :+1: from me.

File: _posts/2016-04-22-next-cio.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+published: true
+layout: post
+author: Adam Scott and Ashwin Vasan
+title: "We're searching for our next CIO"
```
Comment (marcesher) [inline _posts/2016-04-22-next-cio.md]: 👍

File: _posts/2016-04-22-next-cio.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+permalink: /articles/searching-for-our-next-cio/
+---
+
+![desk illustration](/img/cio/CIO_illustration.png)
+
+*The CFPB is looking for our next Chief Information Officer (CIO) to lead our technology team in supporting the mission of the CFPB and in setting an exceptional example for how technology can work in government.*
```
Comment (marcesher) [inline _posts/2016-04-22-next-cio.md]: Thank you for adding this call-to-action right up front

File: _posts/2016-04-22-next-cio.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+## Position Overview
+
+The CIO is the CFPB’s senior leader for all aspects of technology. The role is a hybrid of a traditional CIO, providing services that support the Bureau’s operations, and a CTO, responsible for the technology that amplifies the mission effectiveness of the CFPB.
+
+The CIO oversees the following 8 functions, all of which support the Bureau’s mission or operations:
+
+- **Design:** Design of digital products on [consumerfinance.gov](http://consumerfinance.gov), as well as the design of regulatory products such as mandated disclosures and booklets.
+- **Development:** Software engineering for digital products on [consumerfinance.gov](http://consumerfinance.gov) and custom applications for internal use.  Applications on consumerfinance.gov include consumer-facing applications (e.g., [Owning a Home](http://www.consumerfinance.gov/owning-a-home)) and regulatory applications (e.g., [eRegulations](http://www.consumerfinance.gov/eregulations/), [HMDA](http://www.consumerfinance.gov/hmda/) intake).
```
Comment (marcesher) [inline _posts/2016-04-22-next-cio.md]: I believe "HMDA intake" is incorrect here. We do not currently do hmda intake on consumerfinance.gov. Perhaps simply remove the word "intake"

**Comment by ascott1** [comment]
Typos are addressed.

I have feelings about semantic line breaks in markdown that are not in line with the general consensus here. 😅

**Comment by ascott1** [comment]
Thanks for the quick and timely reviews @marteki @higs4281 @marcesher! They are greatly appreciated.

If you think this is good to go, please merge!

**Pull request merged**